numpy/core/src/umath/simd.inc.src

```diff
@@ -3697,4 +3739,113 @@ sse2_@kind@_BOOL(@type@ * op, @type@ * ip, const npy_intp n)
 
 #endif /* NPY_HAVE_SSE2_INTRINSICS */
 
+#ifdef NPY_HAVE_NEON
+/**begin repeat
+ *  #type = npy_float, npy_double#
+ *  #TYPE = FLOAT, DOUBLE#
+ *  #scalarf = npy_sqrtf, npy_sqrt#
+ *  #Nvtype128 = float32x4_t, float64x2_t#
+ *  #Nvtype64 = float32x2_t, float64x1_t#
+ *  #Nvpre = v, v#
```

Comment:
This seems a bit silly, why not just use `v` below?

Reply:
Silly indeed. Removed for universal intrinsics.